Libraries/LogBox/Data/__tests__/parseLogBoxLog-test.js

```diff
@@ -956,8 +956,12 @@ Please follow the instructions at: fburl.com/rn-remote-assets`,
             fileName: '/path/to/filename.js',
             location: {column: 1, row: 1},
           },
-          // TODO: we're missing the second component,
-          // because stackframe-parser does not recognize it.
+          {
```

Comment:
`dtslint` package upgrades the version of `stacktrace-parser` which changes the behavior for this test. Confirmed with @rickhanlonii it's good to update these snapshots